presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergTableHandle.java

```diff
@@ -90,6 +97,12 @@ public TableType getTableType()
         return predicate;
```

Comment:
We should also make a change to the function call `icebergTableHandle.getPredicate()` in `TableStatisticsMaker::makeTableStatistics`?